Makefile

```diff
@@ -16,6 +16,16 @@ test:
 	./node_modules/.bin/eslint bouncer/scripts
 	./node_modules/karma/bin/karma start karma.config.js
 
+.PHONY: coverage
+coverage: .coverage
+	@pip install -q tox
+	tox -e coverage
+
+.PHONY: codecov
+codecov: .coverage
```

Comment:
Ok, so if you wanted to fix that you'd create a rule for how to create the `.coverage` file. In fact, you've already written a task (`test`) that creates `.coverage`, so adding that rule is a one-liner. Perhaps see if you can work out how you'd add that, otherwise come poke me on Slack!